angular_bind_polymer.js

```diff
@@ -2,6 +2,7 @@ angular.module('eee-c.angularBindPolymer', []).
 directive('bindPolymer', function() {
   return {
     restrict: 'A',
+    scope: {}, /* Scope added here */
```

Comment:
Callback support seems reasonable. Maybe rework the PR so that `index.html` has a native HTML element working correctly with variable _and_ callback binding, then do the same with the `<x-double>` Polymer element where it does not work correctly.

I think we should not make any changes to `angular_bind_polymer.js` — at least not yet — so let's revert this scope line. Once we have a solid test case, then we can make changes to it in order to support callbacks. Sound good?